crt_portal/cts_forms/templates/forms/widgets/multi_select_option.html

```diff
@@ -1,3 +1,3 @@
 <option value="{{option.value}}" {% if option.selected is not False %}selected{% endif %}>
-  {{option.value|title}}
+  {{option.value|default:option.label|title}}
```

Comment:
Required because otherwise we always default to a blank value. Note that this impacts other forms in proforms as well.